icu4c/source/tools/ctestfw/tstdtmod.cpp

```diff
@@ -88,12 +111,14 @@ void IcuTestErrorCode::setScope(const UnicodeString& message) {
 }
 
 void IcuTestErrorCode::handleFailure() const {
-    errlog(FALSE, nullptr);
+    errlog(FALSE, u"", nullptr);
 }
 
-void IcuTestErrorCode::errlog(UBool dataErr, const char* extraMessage) const {
+void IcuTestErrorCode::errlog(UBool dataErr, UnicodeString mainMessage, const char* extraMessage) const {
```

Comment:
Please use a const reference for mainMessage.

Reply:
Done.